documentation/src/docs/asciidoc/release-notes/release-notes-5.4.0-M2.adoc

```diff
@@ -24,11 +24,14 @@ repository on GitHub.
 
 ==== Deprecations and Breaking Changes
 
-* ❓
+* `TestPlan.add(TestIdentifier)` was always considered internal but was missing
+  a corresponding `@API` annotation.
```

Comment:
`TestPlan.add(TestIdentifier)` has always been considered _internal_ and is now appropriately annotated with `@API(status = INTERNAL)`.